src/components/Controls.vue

```diff
@@ -1,28 +1,20 @@
 <script>
 import NodeLink from '@/components/NodeLink/NodeLink.vue';
+import Legend from '@/components/NodeLink/Legend.vue';
 
 import { setUpProvenance } from "@/lib/provenance";
 import { getUrlVars } from "@/lib/utils";
 import { loadData } from "@/lib/multinet";
 
-/**
- * Demo Controls.  This component is meant to demo
- * the capabilities of the NodeLink vis.
- *
- * You'll probably want the controls to look or
- * work differently in your app, so it's recommended that you
- * implement the controls yourself.
- */
+import * as d3 from "d3";
```

Comment:
You will definitely want to import things from d3 by name going forward.

Reply:
Opened #70